resources/migrations/20211221074525-user-fleet.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE user_fleet(
```

Comment:
A few things to note:
- What is the primary key here?
- We should add `created_at` and `updated_at` columns for future auditing purposes
- Pluralize the table name

Reply:
Will implement #2 & #3 points.

Regarding #1 primary key, do we really need it?

Reply:
Will add a compound primary key